### PR TITLE
[LB] Update Least Outstanding Requests Steering descriptions

### PR DESCRIPTION
**Pull request by tc80**
The descriptions introduced by https://github.com/cloudflare/cloudflare-docs/pull/9455 need to be re-worked. 

The current descriptions are confusing since one definition is used to describe LORS at the LB-level and LORS at the pool-level. Although the steering logic is similar, Load Balancers and Pools are very different concepts and can't be mixed. There's a few other issues, such as using the word "connections" rather than "requests" (which would describe a completely different steering method! 😉 ). Also, we have decided to go with `Least Outstanding Requests Steering` with an `s`, so some minor changes there.

Note: I have not previewed this change yet, so I still need to double-check all the links are working properly.

### CODE REVIEW

File: content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  title: Least Outstanding Requests steering
+---
+
+# Least Outstanding Requests steering
+
+**Least Outstanding Requests Steering** allows you to route traffic to origins that currently have the fewest number of outstanding requests.
+
+This steering policy selects an origin by taking into consideration origin weights, as well as each origin's number of in-flight requests. Origins with more pending requests are weighted proportionately less in relation to others.
+
+Least Outstanding Requests Steering is best to use if your origins can be easily overwhelmed by a spike in concurrent requests. Alike other steering policies, it supports [adaptive routing](/load-balancing/understand-basics/adaptive-routing/) and [session affinity](/load-balancing/understand-basics/session-affinity/).
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md]:
```suggestion
Least Outstanding Requests steering is best to use if your origins are easily overwhelmed by a spike in concurrent requests. It supports [adaptive routing](/load-balancing/understand-basics/adaptive-routing/) and [session affinity](/load-balancing/understand-basics/session-affinity/).
```

File: content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+{
+  "origin_steering": {
+    "policy": "least_outstanding_requests"
+  }
+}
+```
+
+Refer to the [API documentation](/api/operations/load-balancer-pools-update-pool) for more information on the pool configuration.
+
+Note that Least Outstanding Requests Steering can also be configured on a load balancer as a [steering policy](/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests/), taking into account outstanding request counts and `random_steering` weights for pools on the load balancer.
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md]:
```suggestion
{{<Aside type="note">}}
Least Outstanding Requests steering can also be configured on a load balancer as a [steering policy](/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests/), taking into account outstanding request counts and `random_steering` weights for pools on the load balancer.
{{</Aside>}}
```

File: content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Limitations
+
+Least Outstanding Requests Steering can be configured for pools that are part of [DNS-only load balancers](/load-balancing/understand-basics/proxy-modes/#dns-only-load-balancing), but is only supported in a no-operation, dummy form. When origin steering logic is applied for a pool on a DNS-only load balancer, all origin outstanding request counts are considered to be zero, meaning traffic is served solely based on origin weights.
+
+Although it is configurable, it is not recommended to associate pools that use Least Outstanding Requests Steering with DNS-only load balancers due to its partial support.
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md]:
```suggestion
Although it is configurable, it is not recommended to associate pools that use Least Outstanding Requests steering with DNS-only load balancers due to its partial support.
```

File: content/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  title: Least Outstanding Requests steering
+---
+
+# Least Outstanding Requests steering
+
+**Least Outstanding Requests Steering** allows you to route traffic to pools that currently have the fewest number of outstanding requests.
+
+This steering policy selects a pool by taking into consideration `random_steering` weights, as well as each pool's number of in-flight requests. Pools with more pending requests are weighted proportionately less in relation to others.
+
+Least Outstanding Requests Steering is best to use if your pools can be easily overwhelmed by a spike in concurrent requests. This steering method lends itself to applications that value server health above latency, geographic alignment, or other metrics. Alike other steering policies, it takes into account the [pool's health status](/load-balancing/understand-basics/health-details/#how-a-pool-becomes-unhealthy), [adaptive routing](/load-balancing/understand-basics/adaptive-routing/), and [session affinity](/load-balancing/understand-basics/session-affinity/).
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests.md]:
```suggestion
Least Outstanding Requests Steering is best to use if your pools are easily overwhelmed by a spike in concurrent requests. This steering method lends itself to applications that value server health above latency, geographic alignment, or other metrics. It takes into account the [pool's health status](/load-balancing/understand-basics/health-details/#how-a-pool-becomes-unhealthy), [adaptive routing](/load-balancing/understand-basics/adaptive-routing/), and [session affinity](/load-balancing/understand-basics/session-affinity/).
```

File: content/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+header: Load Balancers
+---
+{
+  "steering_policy": "least_outstanding_requests"
+}
+```
+
+Refer to the [API documentation](/api/operations/load-balancers-update-load-balancer) for more information on the load balancer configuration.
+
+Note that Least Outstanding Requests Steering can also be configured on a pool as an [origin steering policy](/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools/), taking into account outstanding request counts and weights for origins within the pool.
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests.md]:
```suggestion
{{<Aside type="note">}}
Least Outstanding Requests Steering can also be configured on a pool as an [origin steering policy](/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools/), taking into account outstanding request counts and weights for origins within the pool.
{{</Aside>}}
```

File: content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+}
+```
+
+Refer to the [API documentation](/api/operations/load-balancer-pools-update-pool) for more information on the pool configuration.
+
+Note that Least Outstanding Requests Steering can also be configured on a load balancer as a [steering policy](/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests/), taking into account outstanding request counts and `random_steering` weights for pools on the load balancer.
+
+## Limitations
+
+Least Outstanding Requests Steering can be configured for pools that are part of [DNS-only load balancers](/load-balancing/understand-basics/proxy-modes/#dns-only-load-balancing), but is only supported in a no-operation, dummy form. When origin steering logic is applied for a pool on a DNS-only load balancer, all origin outstanding request counts are considered to be zero, meaning traffic is served solely based on origin weights.
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md]:
```suggestion
Least Outstanding Requests steering can be configured for pools that are part of [DNS-only load balancers](/load-balancing/understand-basics/proxy-modes/#dns-only-load-balancing), but is only supported in a no-operation, dummy form. When origin steering logic is applied for a pool on a DNS-only load balancer, all origin outstanding request counts are considered to be zero, meaning traffic is served solely based on origin weights.
```

File: content/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+## Limitations
+
+Least Outstanding Requests Steering can be configured for [DNS-only load balancers](/load-balancing/understand-basics/proxy-modes/#dns-only-load-balancing), but is only supported in a no-operation, dummy form. For DNS-only load balancers, all pool outstanding request counts are considered to be zero, meaning traffic is served solely based on `random_steering` weights.
+
+Although it is configurable, it is not recommended to use Least Outstanding Requests Steering for DNS-only load balancers due to its partial support.
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/steering-policies/least-outstanding-requests.md]:
```suggestion
Least Outstanding Requests steering can be configured for [DNS-only load balancers](/load-balancing/understand-basics/proxy-modes/#dns-only-load-balancing), but is only supported in a no-operation, dummy form. For DNS-only load balancers, all pool outstanding request counts are considered to be zero, meaning traffic is served solely based on `random_steering` weights.

Although it is configurable, it is not recommended to use Least Outstanding Requests steering for DNS-only load balancers due to its partial support.
```

File: content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md (hunk 1 above)
Comment (tc80) [reply content/load-balancing/understand-basics/traffic-steering/origin-level-steering/least-outstanding-requests-pools.md]: In the dev docs all of the titles have Steering lowercased in the title but capitalized in the text. I stayed consistent with that, but if you want to make this change perhaps we should make it for all of the policies. 

This is the case for Geo, Dynamic, and Proximity steering. (Standard steering has it lowercased in the text).

**Review by patriciasantaana — APPROVED** [review]

**Pull request merged**